frontend/integration-tests/tests/monitoring.scenario.ts

```diff
@@ -214,7 +214,7 @@ describe('Alertmanager: YAML', () => {
   });
 });
 
-describe('Alertmanager: Configuration', () => {
+xdescribe('Alertmanager: Configuration', () => {
```

Comment:
I think the tests above this one in this scenario are OK

Reply:
Ok, fixed

Reply:
@dtaylor113 sorry, I mean to leave `xdescribe` here, it's just that you disabled several tests that were passing above this point in the file.

Reply:
Ok, last change/commit isolates failing tests

Reply:
I don't think it's a good idea to comment out parts of a test. We should disable it with the `x` prefix until we can fix. This makes it clear that the test is disabled and makes it clear what we need to go back and reenable.

Reply:
Ok, updated again